package.json

```diff
@@ -29,6 +31,7 @@
     "json-e": "2.2.1",
     "json-schema-defaults": "0.3.0",
     "lodash": "4.17.4",
+    "marked": "^0.3.6",
```

Comment:
This dependency isn't used directly in treeherder - `angular-marked` specifies `marked` as a devDependency itself (which will be pinned in `yarn.lock`). I'll remove this in another PR that's touching packing soon.

Reply:
> I'll remove this in another PR that's touching packing soon.

Handled in #2840 :-)